cwlavro-tools/src/main/java/io/cwl/avro/CWL.java

```diff
@@ -38,11 +52,60 @@
  */
 public class CWL {
 
+    public static final String RABIX_VERSION = "1.0.0-rc5";
+    public static final String RABIX_GITHUB_LOCATION = "https://github.com/rabix/bunny/releases/download/v"+RABIX_VERSION+"/rabix-"+RABIX_VERSION+".tar.gz";
+    public static final String RABIX_EXEC_LOCATION = ".dockstore/libraries/rabix-"+RABIX_VERSION+"/rabix-backend-local-"+RABIX_VERSION+"/rabix";
+
     private final Gson gson;
     private static final Logger log = LoggerFactory.getLogger(CWL.class);
```

Comment:
I don't think we have a checkstyle file setup for this dependency. 
That said, I think the formatting matches Dockstore already https://github.com/ga4gh/dockstore/blob/develop/dockstore-client/src/main/java/io/dockstore/client/cli/Client.java#L88